tests/sentry/utils/kvstore/test_common.py

```diff
@@ -0,0 +1,93 @@
+import itertools
+import pytest
+from dataclasses import dataclass
+from typing import Iterator, Tuple
+
+from sentry.utils.kvstore.abstract import K, KVStorage, V
+
+
+@dataclass
+class Properties:
```

Comment:
I feel like there has to be a better name for this but I'm drawing a blank on what it would be.